local-tests/tests/wrapped-keys/util.ts

```diff
@@ -1,7 +1,50 @@
 import { LIT_NETWORKS_KEYS } from '@lit-protocol/types';
 import { LIT_CHAINS } from '@lit-protocol/constants';
 import { ethers } from 'ethers';
-import { EthereumLitTransaction } from '@lit-protocol/wrapped-keys';
+import { config } from '@lit-protocol/wrapped-keys';
+import {
+  litActionRepositoryCommon,
+  litActionRepository,
+} from '@lit-protocol/wrapped-keys-lit-actions';
+
+import type {
+  LitActionCodeRepository,
+  LitActionCodeRepositoryCommon,
+  EthereumLitTransaction,
+} from '@lit-protocol/wrapped-keys';
+
+const emptyLitActionRepositoryCommon: LitActionCodeRepositoryCommon = {
```

Comment:
The way the fallback logic inside of the `wrapped-keys` package works, if the code for a particular LIT action is an empty string (falsey), it uses CIDs -- otherwise it uses the string as the LIT action code. It's 'empty' because it contains no code at all, for any key -- basically it resets the entire code repository so that any calls to use the LIT actions will use the defined CIDs instead of trying to use code.  For type safety it needs to be an object with the correct keys, where all appropriate values remain strings.

Reply:
Keeping the values on the object always as strings keeps the type consistent so handling values in the object requires less complexity in typescript -- because values on the object are all of type `string`, we always know what type a value is without needing to do assertions against it, and the code to check if it's a falsy string to know if we should try to use any particular value as code or not is the same as if the value was 'null'  :)